genref/config.yaml

```diff
@@ -116,6 +116,11 @@ apis:
     package: k8s.io/apiserver
     path: pkg/apis/audit/v1
 
+  - name: apiserver-resourcequota
+    title: kube-apiserver ResourceQuota Config (v1)
```

Comment:
nit: The other title strings include "Configuration" not "Config".

Reply:
Fixed.